### PR TITLE
Refactor examples

### PR DESCRIPTION
**Pull request by shoota**
# TL;DR

- fix dead links and using relative path.
- set correct breadcrumb string
- fix path between production and dev-server

### How to check
- checkout this pr
- yarn install && yarn build
- npm start
- open [examples](http://localhost:8080/agent.js/simple/)

### Check list
- [x] not be dead-link in all documents.
- [x] `yarn e2e` finished with success.

### CODE REVIEW

File: examples/webpack.config.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ const path = require('path')
 module.exports = {
   devServer: {
     contentBase: path.join(__dirname, 'build'),
-    openPage: '/simple/'
+    openPage: '/simple/',
+    publicPath: '/agent.js/'
```
Comment (shoota) [inline examples/webpack.config.ts]: gh-page path has `/agent.js`

File: package.json [hunk 2]
```diff
@@ -102,8 +102,8 @@
     "release": "lerna exec --bail=false -- 'can-npm-publish && npm publish'",
     "start": "concurrently -k 'yarn w:examples' 'yarn w:website' 'yarn w:html'",
     "test": "karma start",
-    "w:examples": "webpack-dev-server --config examples/webpack.config.ts --output-path examples/build/assets --env production",
```
Comment (shoota) [inline package.json]: unused option

File: package.json [hunk 3]
```diff
@@ -102,8 +102,8 @@
...
-    "w:html": "pug examples -o examples/build -P -s -w",
+    "w:examples": "webpack-dev-server --config examples/webpack.config.ts --env production",
+    "w:html": "pug examples -o examples/build/agent.js -P -s -w",
```
Comment (shoota) [inline package.json]: fix with webpack-dev-server

**Comment by shkumagai** [comment]
Blogのリンクが404になるのはOKなんでしたっけ？

File: examples/built-in/index.pug [hunk 4]
```diff
@@ -3,5 +3,8 @@ extends ../index.pug
 block content
   include:markdown-it README.md
 
+block breadcrumb-current
+  li.active use with nodejs
```
Comment (Surgo) [inline examples/built-in/index.pug]: Node.js® ?

**Pull request merged**